alpaca/broker/client.py

```diff
@@ -589,3 +597,114 @@ def download_trade_document_for_account_by_id(
+
+        if not isinstance(
+            ach_data, (CreateACHRelationshipRequest, CreatePlaidRelationshipRequest)
+        ):
+            raise ValueError(
+                f"Request data must either be a CreateACHRelationshipRequest instance, or a "
+                f"CreatePlaidRelationshipRequest instance. Got unsupported {type(ach_data)} instead."
+            )
+
+        response = self.post(
+            f"/accounts/{account_id}/ach_relationships", ach_data.to_request_fields()
+        )
+        return ACHRelationship(**response)
+
+    def get_ach_relationships_for_account(
+        self,
+        account_id: Union[UUID, str],
+        statuses: Optional[List[ACHRelationshipStatus]] = None,
+    ) -> List[ACHRelationship]:
+        account_id = validate_uuid_id_param(account_id)
+
+        params = {}
+        if statuses is not None:
+            params["statuses"] = ",".join(statuses)
+
+        response = self.get(f"/accounts/{account_id}/ach_relationships", params)
+        return parse_obj_as(List[ACHRelationship], response)
+
+    def delete_ach_relationship_for_account(
+        self,
+        account_id: Union[UUID, str],
+        ach_relationship_id: Union[UUID, str],
+    ) -> None:
+        account_id = validate_uuid_id_param(account_id)
+        ach_relationship_id = validate_uuid_id_param(ach_relationship_id)
+        self.delete(f"/accounts/{account_id}/ach_relationships/{ach_relationship_id}")
+
+    def create_bank_for_account(
+        self,
+        account_id: Union[UUID, str],
+        bank_data: CreateBankRequest,
+    ) -> Bank:
+        account_id = validate_uuid_id_param(account_id)
+        response = self.post(
+            f"/accounts/{account_id}/recipient_banks", bank_data.to_request_fields()
+        )
+        return Bank(**response)
+
+    def get_banks_for_account(
+        self,
+        account_id: Union[UUID, str],
+    ) -> List[Bank]:
+        account_id = validate_uuid_id_param(account_id)
+        response = self.get(f"/accounts/{account_id}/recipient_banks")
+        return parse_obj_as(List[Bank], response)
+
+    def delete_bank_for_account(
+        self,
+        account_id: Union[UUID, str],
+        bank_id: Union[UUID, str],
+    ) -> None:
+        account_id = validate_uuid_id_param(account_id)
+        bank_id = validate_uuid_id_param(bank_id)
+        self.delete(f"/accounts/{account_id}/recipient_banks/{bank_id}")
+
+    def create_transfer_for_account(
+        self,
+        account_id: Union[UUID, str],
+        transfer_data: CreateTransferRequest,
+    ) -> Transfer:
+        account_id = validate_uuid_id_param(account_id)
+        response = self.post(
+            f"/accounts/{account_id}/transfers", transfer_data.to_request_fields()
+        )
+        return Transfer(**response)
+
+    def get_transfers_for_account(
+        self,
+        account_id: Union[UUID, str],
+    ) -> List[Transfer]:
+        # TODO: Filter params?
```

Comment:
The transfers endpoint (https://alpaca.markets/docs/api-references/broker-api/funding/transfers/#retrieving-all-transfers-by-account) supports some filtering as well as pagination, but the spec in #32 did not include this, so I ignored it for now. The filtering should be simple to add, but pagination can be quite the task!

Reply:
Oh sorry :see_no_evil:. Yeah if there are anything wrong in the spec please call them out they aren't really concrete requirements. I think the filtering is definitely required for this PR if you can please add it in :pray:.  


I definitely forgot about pagination :disappointed:, We do have an example of the pagination in `get_account_activities` if you're looking for a reference but if not I am also okay with creating another ticket to come back and add the pagination here :thinking:

Reply:
Do you have any additional documentation on the pagination for transfers? It seems to have been implemented slightly different than the one for account activities. Mostly blocked by what the default `limit` is for this endpoint.

Reply:
Oh man, to be entirely honest I didn't realize this endpoint doesn't follow our normal model of the page token setup 🤦.

I think we can take generic approach would be to just increment offset by the length of the last response until you get an error or an empty array. Testing locally it looks like requesting an offset that is outside the set of results returns empty array.